### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by iamtheru**
- [DEMO LINK](https://iamtheru.github.io/layout_style-it-up/) <br>
- [TEST REPORT LINK](https://iamtheru.github.io/layout_style-it-up/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,28 +1,28 @@
...
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Document</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <div id="header"></div>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
```
Comment (irenhh) [inline src/index.html:6]:
```suggestion
    <meta 
      name="viewport"
      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
    >
```

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,41 @@
+body {
+  margin: 0;
+}
+
+.important,
+.interesting,
+.surprising,
+div:last-child,
+div:first-child {
```
Comment (irenhh) [inline src/style.css:9]: how about just adding one more class and use with those elements you need?

**Review by irenhh — CHANGES_REQUESTED** [review]

File: src/style.css (hunk 2 above)
Comment (iamtheru) [reply src/style.css:9]: image

Comment (iamtheru) [reply src/style.css:9]: In task, editing index.html are prohibited, or I misunderstood?

Comment (irenhh) [reply src/style.css:9]: adding new class will not break anything, but ok, in this case let it be as is

File: src/index.html [hunk 3]
```diff
@@ -1,28 +1,31 @@
...
-</head>
-<body>
-  <div id="header"></div>
+  <head>
+    <meta charset="UTF-8">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no,initial-scale=1.0,
+      maximum-scale=1.0, minimum-scale=1.0"
+    >
```
Comment (irenhh) [inline src/index.html:9]:
```suggestion
    <meta
      name="viewport"
      content="width=device-width, user-scalable=no,initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
    >
```
move just new attributes

**Review by irenhh — APPROVED** [review]

**Pull request closed without merging**